docs/features.md

```diff
@@ -2,4 +2,35 @@ Click the below for a walk through of the product functionality:
 
 [![PostHog demo](_media/posthog-demo.png)](https://youtu.be/vGy9gzMAAes-Y?t=35s "PostHog Demo Video")
 
-There is a detailed table of the functionality on the [pricing](/pricing) page.
\ No newline at end of file
+There is a detailed table of the functionality on the [pricing](/pricing) page.
+
+Below is a table of PostHog features and how this compares with the functionality in common alternatives.
+
+
+|  Feature 	|   PostHog	| Amplitude  	|  Mixpanel 	| Heap  	|
+|:-:	|:-:	|:-:	|:-:	|:-:	|
+|   Autocapture	|   ✔	|   	|   	|   ✔	|
+|   API 	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   In-App Events Tracking	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Conversion Tracking	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Dashboard	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Data Visualisation	|  ✔ 	|   ✔	|   ✔	|   	|
+|   Cohort Analysis	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Funnel Analysis	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Push Notifications	|   	|   ✔	|   ✔	|   ✔	|
+|   Retention Tracking	|   	|   ✔	|   ✔	|   ✔	|
```

Comment:
We have some of this with Stickiness. Would be good to split this out if possible so we can see exactly what we're missing

Reply:
Will look into this further

Reply:
I think Retention tracking (at least from Mixpanel and Amplitude) specifically means the ability to select the actions that involve a retained user and the time period over which retention is being measured which outputs something like this:


Cohort | No. of | < 1 Day | 2 Days | 3 Days | 4 Days | 5 Days
-- | -- | -- | -- | -- | -- | --
New Users | 52 | 97% | 44% | 36% | 30% | 20%
Super User | 13 | 88% | 58% | 46% | 25% | 18%

With some form of conditional formatting applied.

I will add User Engagement metrics - which aligns well with the powerful nature of Trends already